src/graphql/main/schema.graphql

```diff
@@ -974,6 +975,14 @@ type Query {
     @deprecated(
       reason: "TODO: remove. we don't need a non authenticated version of this query. the users can only do the query while authenticated"
     )
+
+  """
+  Returns 1 Sat or 1 Cent price for the given quote display currency
+  """
+  realtimePrice(
```

Comment:
will also need the type on the subscription

Reply:
Updated https://github.com/GaloyMoney/galoy/commit/087c47c24422414876cf60ae7ab55bf1e00bb51c